spec/contracts/notifications/create_contract_spec.rb

```diff
@@ -73,46 +67,17 @@
     end
 
     context 'without a reason for IAN with read_ian false' do
-      let(:notification_reason_ian) { nil }
+      let(:notification_reason) { nil }
       let(:notification_read_ian) { false }
 
-      it_behaves_like 'contract is invalid', reason_ian: :no_notification_reason
+      it_behaves_like 'contract is invalid', reason: :no_notification_reason
     end
 
     context 'without a reason for IAN with read_ian nil' do
```

Comment:
This spec is wrong now, since there should always be a reason, as checked by the spec above.